tests/grafana_addon_test.go

```diff
@@ -46,7 +50,52 @@ func TestHelmGrafanaAddon(t *testing.T) {
 	}
 
 	waitUntilServicesAvailable(t, *helmOptions.KubectlOptions, []string{"grafana"})
+	waitUntilDeploymentsAvailable(t, *helmOptions.KubectlOptions, []string{"grafana"})
+
+	// valuesFile, err := os.ReadFile("../addons/grafana/values.yaml")
+
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+
+	// yamlData := make(map[string]interface{})
+
+	// err = yaml.Unmarshal(valuesFile, &yamlData)
+
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+
+	// grafanaSource, ok := yamlData["grafana-source"].(map[string]interface{})
+	// if !ok {
+	// 	log.Fatalf("Error accessing 'grafana-source' key")
+	// }
+
+	// adminPassword, ok := grafanaSource["adminPassword"].(string)
+	// if !ok {
+	// 	log.Fatalf("Error accessing 'adminPassword' key")
+	// }
 
+	grafanaKubectlOptions := k8s.NewKubectlOptions("", "", "grafana")
+
+	tunnel := k8s.NewTunnel(grafanaKubectlOptions, k8s.ResourceTypeService, "grafana", 0, 3000)
```

Comment:
Few improvements

- can we put "grafana" in a constant so we don't repeat the string over and over?
- can we read the port from the configuration?